src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -238,6 +242,58 @@ Similarly, the OPTIONAL [`rec-<label>`](../99-appendices/09-entities.md#rec)
+| Magnetization transfer ratio map             | MTRmap    | In percentage (%). MTR maps are REQUIRED to use this suffix irrespective of the method they are related to                                                                                                                                                                                                                                                                                                                                                                           |
+| Magnetization transfer saturation map        | MTsat     | In arbitrary units (a.u.). MTsat maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                          |
+| Homogeneous (flat) T1-weighted MP2RAGE image | UNIT1     | In arbitrary units (a.u.). UNIT1 images are REQUIRED to use this suffix irrespective of the method they are related to. Note that although this image is T1-weighted, regions without MR signal will contain white salt-and-pepper noise that most segmentation algorithms will fail on. Therefore, it is important to dissociate it from from `T1w`.                                                                                                                                                        |
+| T1 in rotating frame (T1 rho) map            | T1rho     | In seconds (s). T1-rho maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                          |
+| Myelin water fraction map                    | MWFmap    | In percentage (%). MWF maps are REQUIRED to use this suffix irrespective of the method they are related to.                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
We do not have "percentage" supported in the specification yet. I'm not sure if we'll want to use symbols for abbreviations.

Reply:
What would be a good way to represent images with fractional intensity values? Can we add `percent` to the units and not use symbols?

Reply:
Can we just say arbitrary units? I mean ratio and percentage are not units.

Reply:
I think that's probably the way to go, although I think `pct` would also be reasonable.

Reply:
@tiborauer `pct` would be nice to unify the choice between 0-1 vs 0-100. Even they are not units, having a convention for fractional scales can be useful.

Reply:
What does, say, `10 pct` would tell us? Is it interpretable/comparable across measurements/participants? I understand that it reflects the process of how it was calculated. 
On the other hand, I agree that the choice between 0-1 vs 0-100 should be explicit. But it does not require a unit.

Reply:
I think it will require a unit if the values are going to be validated automatically at some point.